lib/ReactViews/Map/Panels/ToolsPanel/tools-panel.scss

```diff
@@ -18,3 +18,7 @@
 .results {
   padding: 7px;
 }
+
+.dropdown-inner {
```

Comment:
Adding more scss, probably better to not waste time on making styled component hey

Reply:
I didn't want to spend time on ToolsPanel, since it's only intended to be used by us to e.g. count datasets in the catalogue, but I guess the maintenance is still just as costly. Hopefully a component like this will be super easy to make with Chakra.